src/parse/stmt.rs

```diff
@@ -262,6 +267,7 @@ impl<I: Lexer> Parser<I> {
     }
 
     /// expr_opt: expr ';' | ';'
```

Comment:
```suggestion
    /// `expr_opt: expr ';' | ';'`
    ///
```
Before: 
![image](https://user-images.githubusercontent.com/23638587/81767421-62207e00-94a6-11ea-80c5-a1678b637d5d.png)
After: 
![image](https://user-images.githubusercontent.com/23638587/81767509-9300b300-94a6-11ea-8798-44ed2643f951.png)